### PR TITLE
tidb-control: modify --port to status port

### PR DESCRIPTION
**Pull request by sultan8252**
Modify --port to status port
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by sultan8252 — COMMENTED** [review]
PTAL

**Review by lonng — COMMENTED** [review]
/lgtm

File: tidb-control.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多
 `tidb-ctl` 有 4 个与连接相关的全局参数，分别为：
 
 - `--host` TiDB 服务地址
-- `--port` TiDB 服务端口
+- `--port` TiDB status端口
```
Comment (TomShawn) [inline tidb-control.md:62]:
```suggestion
- `--port` TiDB status 端口
```

File: tidb-control.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ tid 子命令用来通过表的 id 获取数据库中表的表结构。
 }
 ``` 
 
-同 in 子命令一样，如果使用的 TiDB 地址不是默认的地址和端口，需要通过 `--host` 和 `--port` 参数指定 TiDB 的地址和端口。
+同 in 子命令一样，如果使用的 TiDB 地址不是默认的地址和端口，需要通过 `--host` 和 `--port` 参数指定 TiDB 的地址和status端口。
```
Comment (TomShawn) [inline tidb-control.md:140]:
```suggestion
同 in 子命令一样，如果使用的 TiDB 地址不是默认的地址和端口，需要通过 `--host` 和 `--port` 参数指定 TiDB 的地址和 status 端口。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**